docs/ref/settings.txt

```diff
@@ -911,6 +911,74 @@ This is an Oracle-specific setting.
 
 The maximum size that the DATAFILE_TMP is allowed to grow to.
 
+.. setting:: DATAFILE_SIZE
+
+``DATAFILE_SIZE``
+^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.0
+
+Default: ``'50M'``
+
+.. versionchanged:: 2.0
+
+    The previous value was 20M and was not user customizable.
+
+This is an Oracle-specific setting.
+
+The initial size of the DATAFILE.
+
+.. setting:: DATAFILE_TMP_SIZE
+
+``DATAFILE_TMP_SIZE``
+^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.0
+
+Default: ``'50M'``
+
+.. versionchanged:: 2.0
+
+    The previous value was 20M and was not user customizable.
+
+This is an Oracle-specific setting.
+
+The initial size of the DATAFILE_TMP.
+
+.. setting:: DATAFILE_EXTSIZE
+
+``DATAFILE_EXTSIZE``
+^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.0
+
+Default: ``'25M'``
+
+.. versionchanged:: 2.0
+
+    The previous value was 10M and was not user customizable.
+
+This is an Oracle-specific setting.
+
+The size that the DATAFILE is extended when more space is required.
```

Comment:
The amount by which the DATAFILE is extended when more space is required.
(use similar wording for next note also)